app/src/main/java/com/example/marvelapp/presentation/mvp/model/MarvelAppModel.kt

```diff
@@ -5,8 +5,13 @@ import com.example.marvelapp.domain.usecase.GetCharacterUseCase
 import com.example.marvelapp.presentation.mvp.contract.MarvelAppContract
 import io.reactivex.rxjava3.core.Observable
 
-class MarvelAppModel(private val getCharacterUseCases: GetCharacterUseCase) : MarvelAppContract.MarvelAppModel {
+class MarvelAppModel(private val getCharacterUseCases: GetCharacterUseCase) :
+    MarvelAppContract.MarvelAppModel {
+    override var charactersList: List<CharacterEntity>
```

Comment:
I don't understand this can you explain me?

Reply:
It was a mistake, in principle it was a method designed for future functionality. Has already been erased